packages/neovim/neovim.pacscript

```diff
@@ -0,0 +1,24 @@
+name="neovim"
+pkgname="neovim"
+version="5.0.0"
+url="https://github.com/neovim/neovim/archive/refs/tags/v0.5.0.tar.gz"
+build_depends="gettext libtool libtool-bin autoconf automake cmake g++ pkg-config unzip build-essential"
+depends=""
+optdepends=()
```

Comment:
No need to add `depends` and `optdepends`, since they aren't used in this script